sdk/src/main/java/io/radar/sdk/Radar.kt

```diff
@@ -760,35 +767,9 @@ object Radar {
             return
         }
 
-        RadarSettings.setListenToServerTrackingOptions(context, false)
         locationManager.startTracking(options)
     }
 
-    /**
```

Comment:
We're not going with the separate method in the SDK anymore. All of the logic is handled in the main startTracking method